gupb/model/coordinates.py

```diff
@@ -14,6 +14,11 @@ def sub_coords(self: Coords, other: Coords) -> Coords:
 def mul_coords(self: Coords, other) -> Coords:
     if isinstance(other, int):
         return Coords(self[0] * other, self[1] * other)
+    # ? mogę tu dodać opcję dla mnozenia przez float? przydaje się czasem ~WS
+    elif isinstance(other, float):
+        return Coords(int(self[0] * other), int(self[1] * other))
+    # * i w ogóle mozna by z tego zrobic porzadna klase, z __add__, __sub__, etc.
```

Comment:
zdecydowanie, to już od dłuższego czasu jest ten moment...albo po prostu przejść na jakąś implementację biblioteczną wektora - natomiast to jest też bardzo bugogenna zmiana i na razie nie miałem serca się za to zabrać